labs-ios-starterUITests/labs_ios_starterUITests.swift

```diff
@@ -0,0 +1,38 @@
+// Copyright © 2020 Shawn James All rights reserved.
+// labs_ios_starterUITests.swift
+
+import XCTest
+
+class labs_ios_starterUITests: XCTestCase {
+
```

Comment:
If we don't need the boilerplate comments in this file, it's good to remove them